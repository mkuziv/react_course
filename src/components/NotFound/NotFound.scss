@import '../../styles/variables.scss';
@import '../../styles/mixin.scss';

.error-page {
  margin-bottom: 10px;
  background: $darkGray;
  height: 396px;
  padding-top: 100px;

  &__wrapper {
    max-width: 960px;
    color: $red;

    .header {
      @include text(50px, 900, 60px);
      text-align: center;
    }
  }
}