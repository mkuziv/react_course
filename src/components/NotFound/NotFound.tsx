import React from 'react';

import './NotFound.scss';

const NotFound = () => (
  <section className="error-page">
    <div className="error-page__wrapper">
      <h3 className="header">PAGE NOT FOUND!</h3>
    </div>
  </section>
);

export default NotFound;
