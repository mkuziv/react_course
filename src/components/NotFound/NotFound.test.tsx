import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import NotFound from './NotFound';

describe('NotFound', () => {
  it('render', () => {
    const { container } = render(<NotFound />);
    expect(container).toMatchSnapshot();
  });
});
