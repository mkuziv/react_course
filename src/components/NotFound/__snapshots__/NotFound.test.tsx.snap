// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound render 1`] = `
<div>
  <section
    class="error-page"
  >
    <div
      class="error-page__wrapper"
    >
      <h3
        class="header"
      >
        PAGE NOT FOUND!
      </h3>
    </div>
  </section>
</div>
`;
